lib/portage_utils.py

```diff
@@ -184,36 +228,50 @@ def open(filename, mode='r', quiet=True, encoding="utf8"):
 
    if filename == "-":
       if mode in ('r', 'rt'):
+         theFile = builtins.open(sys.stdin.fileno(), mode, encoding=encoding, newline=newline)
```

Comment:
`reconfigure` was added to Python 3.7. It would be a nicer choice, but since we don't want to lock the version of python, we have to use this older idiom supported by more versions. `io.TextIOWrapper` also does not work.